docs/adr/0009-non-WMDE-release-notes.md

```diff
@@ -0,0 +1,37 @@
+# 9) Release notes for software not maintained by WMDE {#adr_0009}
+
+Date: 2021-01-15
+
+## Status
+
+proposed
+
+## Context
+
+We intend to package and release software that is is not maintained by WMDE. For example, the Wikidata Query Service (WDQS)
+
+This software comes from a variety of sources including software that is used and maintained by the WMF in Wikimedia
+production Wikis but also some from complete 3rd parties.
+
+Some of this software including ElasticSearch and WikibaseLocalMedia already have curated release notes for versions.
```

Comment:
Not sure if this should be "Some of these"?

Reply:
How would you see it rephrased?

"Some of these software components including ElasticSearch..."?

I think the current phrasing is also grammatically correct (but I'm no expert) because software is a "mass noun" (https://en.wikipedia.org/wiki/Mass_noun) but really I'm just going with my gut :)

Reply:
If you think it's fine the way it is I'm also happy :)